import-service/handlers/importFileParser.ts

```diff
@@ -17,16 +17,21 @@ export const importFileParser: S3Handler = async ({ Records }) => {
 
         s3Stream
           .pipe(csvParser())
-          .on('data', data => {
-            console.log('csv row', data);
+          .on('data', async item => {
+            console.log('csv row', item);
+
+            const sqsSendMessageParams = {
+              QueueUrl: process.env.PRODUCT_SQS_URL!,
```

Comment:
Do you need this '!' after PRODUCT_SQS_URL?

Reply:
Yes, I do because typescript sees the type of `PRODUCT_SQS_URL` as a `string | undefined`